.cirrus.yml

```diff
@@ -123,6 +123,7 @@ build_task:
               CTR_FQIN: ${PRIOR_FEDORA_CONTAINER_FQIN}
               CI_DESIRED_RUNTIME: crun
               CI_DESIRED_NETWORK: cni
+              TMPDIR: /var/tmp
```

Comment:
In case you have to repush please add a comment why we are doing this. I know it is in the git history but once it get moved it will be much harder to figure out at some point in the future.

Reply:
I meant to do that, thanks. Will re-push once I see how CI does.